### DISCUSSION TITLE
Allow users to repeat the alarms

### DISCUSSION THREAD

**Discussion by deepfreelance222** [gdelataillade/alarm#126] [Ideas]
I want to allow users to repeat based on the days chosen by the user. From Monday - Sunday users should be able to choose which days to repeat, also when the notification is clicked, the alarm shouldn't get deleted. Rather it should just stop.

**Comment by gdelataillade** [comment]
Hi @deepfreelance222 

Which version of the plugin are you currently using ?

**Reply by deepfreelance222** [reply]
I'm trying to edit the example project

**Comment by deep526** [comment]
Any update regarding this?

**Reply by gdelataillade** [reply]
Here's the issue associated: #47 
There are some technical limitations, I'm not planning to develop this in the next few months to be honest.
